login-workflow/src/auth-shared/index.ts

```diff
@@ -20,7 +20,7 @@ export * from './contexts/SecurityContextProvider';
 export { AuthUIContextProvider } from './contexts/AuthUIContextProvider';
 export type {
     AuthUIContextProviderProps,
-    RegistrationUIActions,
+    // RegistrationUIActions,
```

Comment:
I have removed RegistrationUIActions from here. that's why the build is failing for example 1. Do I need to update the Build file? @JeffGreiner-eaton , @daileytj

Reply:
Is this the error you see ?

![image](https://github.com/etn-ccis/blui-react-workflows/assets/119693939/00fedba2-2afb-402a-b92f-47c0a2df6cee)

Reply:
Rename  RegistrationUIActions  => RegistrationUIActionsLegacy for example1 https://github.com/etn-ccis/blui-react-workflows/pull/302/commits/fcb5f2dae9044b003f0080af790679182acc4c9a. Build is not failing now. @JeffGreiner-eaton please review it. Thanks!!!